server/tests/test_sse.py

```diff
@@ -86,6 +85,35 @@ async def test_missed_events(minimalorg: MinimalorgRpcClients, backend: Backend)
         )
 
 
+async def test_empty_last_event_id(minimalorg: MinimalorgRpcClients, backend: Backend) -> None:
+    """
+    When the client provide `Last-Event-ID` with an empty value,
+    the server should ignore it and keep processing as if no `Last-Event-ID` was provided.
+    """
+    async with minimalorg.alice.events_listen(last_event_id="") as alice_sse:
+        backend.event_bus._dispatch_incoming_event(  # pyright: ignore[reportPrivateUsage]
+            EventPinged(
+                organization_id=minimalorg.organization_id,
+                ping="recent_event",
+            )
+        )
+
+        # First event is always ServiceConfig
+        event = await alice_sse.next_event()
+        assert event == authenticated_cmds.v4.events_listen.RepOk(
+            authenticated_cmds.v4.events_listen.APIEventServerConfig(
+                active_users_limit=ActiveUsersLimit.NO_LIMIT,
+                user_profile_outsider_allowed=True,
+            )
+        )
+
+        # Backend should inform that some events were missed and we could not retrieve them
+        event = await alice_sse.next_event()
+        assert event == authenticated_cmds.v4.events_listen.RepOk(
+            authenticated_cmds.v4.events_listen.APIEventPinged(ping="recent_event")
+        )
```

Comment:
I don't understand the comment. For me, the backend does not inform of "missed" events (nor that they could not be retrieved). It simply returns events without applying any filtering on event id. 

If that's the case, I would update the comment to something like this:
```suggestion
        # Backend should sent events without applying any filtering on event id 
        event = await alice_sse.next_event()
        assert event == authenticated_cmds.v4.events_listen.RepOk(
            authenticated_cmds.v4.events_listen.APIEventPinged(ping="recent_event")
        )
```

Reply:
It's a wrong copy/paste :sweat_smile: forgot about the comment

Reply:
```suggestion
        # Backend still send new events without processing `Last-Event-ID`
        event = await alice_sse.next_event()
        assert event == authenticated_cmds.v4.events_listen.RepOk(
            authenticated_cmds.v4.events_listen.APIEventPinged(ping="recent_event")
        )
```